app/services/manifest_token_json_converter.rb

```diff
@@ -0,0 +1,22 @@
+class ManifestTokenJsonConverter
```

Comment:
There was some common code essentially copied in a couple of controllers-- it didn't make sense to pull the occasionally used function up to a superclass one-controller-base-class-to-rule-them-all, so it was extracted. It doesn't need to be in the "services" directory and can move to the controllers directory.